etc/profile-m-z/retroarch.profile

```diff
@@ -0,0 +1,46 @@
+# Persistent local customizations
+include retroarch.local
+# Persistent global definitions
+include globals.local
+
+blacklist /boot
+blacklist /opt
+blacklist /usr/libexec
+blacklist /var
```

Comment:
```suggestion
```

Reply:
It works fine without /var.

Reply:
IDC if it works fine for you without `/var` it will not work on systems which use `/var/lib/ca-certificates` for example.

Reply:
Just removed it.